lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -33,6 +70,26 @@ <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
                 <p class="message-copy copy copy-base emphasized">${accomplishment_banner_congrats}</p>
                 <div class="message-actions">
                     <h3 class="sr-only">${_("Take this with you:")}</h3>
```

Comment:
@asadiqbal08 Can I view this in a sandbox, or can you provide a screenshot? I'd like to be clear on the context before I suggest wording. Thanks!